# Service for homing (Resetting the offset) all nodes
---
bool success
